docs/fitting_types.rst

```diff
@@ -385,6 +385,29 @@ are added squared, ready to be minimized. Unlike in the above example, the
 
   Do not cite the overall :math:`R^2` given by :mod:`symfit`.
 
+Fitting moredimensional datasets
+--------------------------------
+Imagine that you have now a specific property over a grid, like the surface height,
+so you have a three-dimensional data set. To fit a function with two parameters
+to this data set, you can use :mod:`symfit`.
+
+At this example, we made a polynomial function with two coefficients,
+representing two terms of mixed order in ``x`` and ``y``::
```

Comment:
Instead of this `::`, you can use a `.. math::` block to first write the equation using latex. With that example in mind, the Poly syntax is more understandable.

Reply:
The version you used is not the newest. I already implemented the mathematical form. 
Please correct me, if I'm wrong :)

Reply:
This problem is done, right? :D